src/pages/trust.js

```diff
@@ -0,0 +1,236 @@
+import React from 'react'
+import { graphql, Link } from 'gatsby'
+
+import '../styles/pages/security-and-trust-center.sass'
+import Banner from '../components/banner'
+import SEO from '../components/seo'
+import Layout from '../components/layout'
+import GetStartedWithTiDBCloud from '../components/get-started-with-TiDB-cloud'
+
+const Trust = React.memo(({ data }) => {
+  const {
+    BannerSVG,
+    DataAccessControlsSVG,
+    DataEncryptionSVG,
+    AuthenticationSVG,
+    WorkloadIsolationSVG,
+    AICPA_SOC_PNG,
+  } = data
+  console.log(data)
+  const className = `SecurityAndTrustCenter`
+  return (
+    <Layout>
+      <SEO
+        title="About PingCAP"
+        description="Story about PingCAP, the team behind TiDB"
+      />
+      <article className="PingCAP-SecurityAndTrustCenter">
+        <Banner backgroundImage={BannerSVG.publicURL}>
+          Security and Trust Center
+        </Banner>
+        <div className={`${className} container`}>
+          <SecurityAndTrustCenterSummary>
+            At PingCAP we value security and trust more than anything because we
+            know our customers entrust us with their most important asset - the
+            data itself. We take this responsibility seriously and are always
+            dedicated to protecting the Security, Availability, and
+            Confidentiality of our customers’ data. We have incorporated
+            security into all aspects of our offering (TiDB Cloud) and
+            operations.
+          </SecurityAndTrustCenterSummary>
+          {/* be careful: use ● instead of the list-style: disc */}
+          <SecurityAndTrustCenterKeySecurityFeatures
+            title={'Key Security Features'}
+            list={[
```

Comment:
Extract data like `[this](https://github.com/pingcap/website/tree/master/src/data)` and import to this page. Use map to apply those data.

Don't use the dot `●` here,  use [bulma list ](https://bulma.io/documentation/elements/content/) to show the list items

Reply:
@cw1997 I  recommend to extract the data into a json file and import to this page.